<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panelExport</class>
 <widget class="QWidget" name="panelExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>79</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="3">
    <widget class="QPushButton" name="buttonExport">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEditUri">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonChooseOutputFile">
     <property name="text">
      <string>Choose output</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEditOutputFile"/>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBoxUri">
     <property name="text">
      <string>Use URI</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxUri</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditUri</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
